middleman/chat_functions.py

```diff
@@ -88,7 +90,16 @@ async def send_text_to_room(
         logger.exception(f"Unable to send message response to {room_id}")
         return f"Failed to send message: {ex}"
 
-
+async def get_dm(client: AsyncClient, user: str): 
```

Comment:
```suggestion

async def get_dm(client: AsyncClient, user: str): 
```

Functions should be separated by two blank lines, see https://pep8.org/#blank-lines